projects/no-os_library_project/RTE/RTE_Components.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Auto generated Run-Time-Environment Component Configuration File
+ *      *** Do not modify ! ***
+ *
+ * Project: no-os
+ * RTE configuration: system.rteconfig
+*/
+#ifndef RTE_COMPONENTS_H
+#define RTE_COMPONENTS_H
+
+/*
+ * Define the Device Header File:
+*/
+#define CMSIS_device_header "ADuCM3029.h"
+
+
```

Comment:
nitpick: extra line

Reply:
This file is generated when creating the project. Modifications shouldn't be done in it

Reply:
Okey then